documentation/modules/ref-address-space-example-exports.adoc

```diff
@@ -5,7 +5,14 @@
 [id='ref-address-space-example-exports-{context}']
 = Address space example exports
 
-This example shows the format used by the `ConfigMap` and `Secret` export types:
+You can export your address space information using the following three export types:
+* `ConfigMap`
+* `Secret`
+* `Service`
+
+== `ConfigMap` and `Secret` type export examples
+
+This example shows the format used by the `ConfigMap` export type. The format of the `Secret` export type uses the same keys as the `ConfigMap` export type, but the values are MD5 encoded.
```

Comment:
My typo, sorry! "MD5" should read "Base64"

Reply:
@k-wall Fixed